independent-projects/arc/runtime/src/main/java/io/quarkus/arc/impl/ArcContainerImpl.java

```diff
@@ -694,7 +695,19 @@ private static Set<InjectableBean<?>> resolve(List<InjectableBean<?>> matching)
         nonDefault.removeIf(InjectableBean::isDefaultBean);
```

Comment:
If you feel adventurous, you could also replace this usage of `removeIf` :laughing:

Reply:
@Ladicek I've added a commit with that as well :)